tests/test_rdsclusterparamgroup.py

```diff
@@ -0,0 +1,43 @@
+from .common import BaseTest
+
+
+class TestRDSParameterGroupFilter(BaseTest):
```

Comment:
this should just be in the `test_rdscluster.py` file

Reply:
done moved test to test_rdscluster.py